similar/editor/med.cpp

```diff
@@ -116,7 +116,7 @@ grs_canvas *Canv_editor_game=&_canv_editor_game; //the game on the editor screen
 
 window *Pad_info;		// Keypad text
 
-grs_font_ptr editor_font;
+grs_font_ptr editor_font = nullptr;
```

Comment:
This should not be necessary.  The default constructor should initialize it to `nullptr` for you.